site/_posts/2019-04-02-0.13.0-release.md

```diff
@@ -0,0 +1,169 @@
+
+The Arrow team is growing! Since the 0.12.0 release we have increased the size
+of our committer and PMC rosters.
+
+* [Andy Grove][13] was promoted to PMC member
+* [Micah Kornfield][14] was added as a committer
+
+Thank you for all your contributions!
+
+## Rust DataFusion Query Engine donation
+
+Since the last release, we received a donation of [DataFusion][17], a
+Rust-native query engine for the Arrow columnar format, whose development had
+been led prior by Andy grove. [Read more about DataFusion][19] in our February
+blog post.
+
+This is an exciting development for the Rust community, and we look forward to
+developing more analytical query processing within the Apache Arrow project.
+
+## Arrow Flight gRPC progress
+
+Over the last couple months, we have made significant progress on Arrow Flight,
+an Arrow-native data messaging framework. We have have integration tests to
+check C++ and Java compatibility, an we have added Python bindings for the C++
+library. We will write a future blog post to go into more detail about how
+Flight works.
+
+## C++ notes
+
+There were 231 issues relating to C++ in this release, far too much to
+summarize in a blog post. Some notable items include:
+
+* An experimental `ExtensionType` was developed for creating user-defined data
+  types that can be embedded in the Arrow binary protocol. This is not yet
+  finalized, but [feedback would be welcome][51].
+* We have undertaken a significant reworking of our CMake build system for C++
+  to make the third party dependencies more configurable. Among other things,
+  this eases work on packaging for Linux distributions. Read more about this in
+  the [C++ developer documentation][18].
+* Laying more groundwork for an Arrow-native in-memory query engine
+* We began building a reader for line-delimited JSON files
+* Gandiva can now be compiled on Windows with Visual Studio
+
+## C# Notes
+
+C# .NET development has picked up since the initial code donation last
+fall. 11 issues were resolved this release cycle.
+
+The Arrow C# package is now available via NuGet.
+
+## Go notes
+
+8 Go-related issues were resolved. A notable feature is the addition of a CSV
+file writer.
+
+## Java notes
+
+26 Java issues were resolved. Outside of Flight-related work, some notable
+items include:
+
+* Migration to Java 8 date and time APIs from Joda
+* Array type support in JDBC adapter
+
+## Python notes
+
+86 Python-related issues were resolved. Some highlights include:
+
+* The Gandiva LLVM expression compiler is now available in the Python wheels
+  through the ``pyarrow.gandiva`` module.
+* Flight RPC bindings
+* Improved pandas serialization performance with RangeIndex
+* pyarrow can be used without pandas installed
+
+Note that Apache Arrow will continue to support Python 2.7 until January 2020.
+
+## Ruby and C GLib notes
+
+36 C/GLib- and Ruby-related issues were resolved. The work continues to follow
+the upstream work in the C++ project.
+
```

Comment:
Ruby has the following notable improvement: 

- `Arrow::RecordBatch#raw_records` was added. It can convert a record batch to a Ruby's array in 10x-200x faster than the same conversion by a pure-Ruby implementation.